db/schema.rb

```diff
@@ -37,8 +37,8 @@
     t.integer "max_cpu_cores"
     t.integer "max_ram_size"
     t.integer "max_storage_size"
-    t.integer "max_shown_vms"
     t.string "git_branch", default: "master"
+    t.integer "max_shown_vms", default: 10
```

Comment:
No migrations in this PR, changes to schema.rb not necessary

Reply:
I removed the changes.